vip_hci/hci_postproc.py

```diff
@@ -16,18 +16,50 @@
 from .metrics import snrmap_fast, snrmap
 from .pca import pca
 import pickle
+import numpy as np
 
 
 class HCIPostProcAlgo(BaseEstimator):
     """ Base HCI post-processing algorithm class.
     """
+    THRESHOLDS_5 = [0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5]
+    THRESHOLDS_1 = np.linspace(0.1, 0.99, 10).tolist()
```

Comment:
We must avoid hard-coding detection thresholds.

Reply:
No, these hard-coded thresholds shouldn't be there

Reply:
✓ removed